src/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/InteropHelpers.cs

```diff
@@ -316,6 +316,37 @@ internal static unsafe void FixupMethodCell(IntPtr hModule, MethodFixupCell* pCe
             }
         }
 
+#if !PLATFORM_UNIX
+        private static unsafe IntPtr GetProcAddress(IntPtr hModule, byte* methodName, CharSet charSetMangling)
+        {
+            var exactMatch = Interop.mincore.GetProcAddress(hModule, methodName);
+
+            if ((charSetMangling == CharSet.Ansi && exactMatch != IntPtr.Zero) || charSetMangling == 0)
```

Comment:
Why is this conditioned on `CharSet.Ansi`?

Reply:
I'm copying the logic from CoreCLR, where the Unicode function takes precedence over the exact match, but the exact match takes precedence over the Ansi function. See [method.hpp in CoreCLR](https://github.com/dotnet/coreclr/blob/master/src/vm/method.cpp#L4925). I've copied the comment from there now.

Reply:
I see. If it matches CoreCLR, it works for me. Thanks for the explanation!

Cc @luqunl since the `FindEntryPoint` in `McgHelpers.cs` on ProjectN doesn't seem to match that behavior.